.github/actions/ngen-build/action.yaml

```diff
@@ -176,6 +180,7 @@ runs:
               -DBMI_FORTRAN_ACTIVE:BOOL=${{ inputs.bmi_fortran }} \
               -DNGEN_ACTIVATE_ROUTING:BOOL=${{ inputs.use_troute }} \
               -DNETCDF_ACTIVE:BOOL=${{ inputs.use_netcdf }} \
+              -DNGEN_WITH_SQLITE:BOOL=${{ inputs.use_sqlite }} \
               -DMPI_ACTIVE:BOOL=${{ inputs.use_mpi }} -S .
```

Comment:
Wouldn't it be in scope to update all of these? Any particular reason why not?

Reply:
The primary reason was to ensure that this PR was entirely backwards compatible -- and ensuring that the deprecated options work as expected (even though they will be removed eventually). This then gives us leeway in updating other portions affected by this PR (i.e. documentation, preprocessor defs, etc.)